tests/js/encryption-tests.js

```diff
@@ -74,10 +81,15 @@ module.exports = {
     },
 
     testEncryptionWithSync: function() {
+        //TODO: remove when MongoDB Realm test server can be hosted on Mac or other options exists
+        if (!isNodeProcess) {
+            return Promise.resolve();;
+        }
```

Comment:
This should probably be listed on https://github.com/realm/realm-js/issues/3312

Reply:
done